fx-src/FirefoxStorage.js

```diff
@@ -108,12 +108,16 @@ const currentSchemaVersion = 1;
 export default class FirefoxAdapter extends BaseAdapter {
   constructor(collection, options={}) {
     super();
+    const {sqliteHandle=null} = options;
     this.collection = collection;
-    this._connection = null;
+    this._connection = sqliteHandle;
     this._options = options;
   }
 
-  _init(connection) {
+  // We need to be capable of calling this from "outside" the adapter
+  // so that someone can initialize a connection and pass it to us in
+  // adapterOptions.
+  static _init(connection) {
```

Comment:
I'd be happier if `_init` had a clearer name that made it obvious that callers needed to use it!